### PR TITLE
README typos

### PR DESCRIPTION
**Pull request by orangemug**
Found some README typos, so decided to write a https://github.com/orangemug/readme-tester and add it to this module, what do you think interested in this PR?

**NOTE:** `readme-tester` probably needs some better error handling / docs before we merge this PR.

### CODE REVIEW

**Comment by orangemug** [comment]
Bumped `readme-tester` to `v0.2.0` ready for review/merge

File: README.md [hunk 1]
```diff
@@ -56,23 +56,25 @@ var validator = hannibal.create({
...
 // Check an invalid user
-validator({
+var basicRslt2 = basicValidator({
     name: "Templeton Peck",
-    age: "38",
+    age: "foo",
     address: {
         city: "Los Angeles"
     }
-}) // {isValid: false, error: {age: {type: "is not a string", address: {street: {required: "is not provided"}}}}, data: {name: "Templeton Peck"}}
```
Comment (oliverbrooks) [inline README.md:75]: I was keen to provide the example response so the reader can see the type of error message.  Any idea how to do this nicely in a message

Comment (orangemug) [inline README.md:75]: We could do a [assert.deepEqual](https://nodejs.org/api/assert.html#assert_assert_deepequal_actual_expected_message) on the `error` message, thoughts?

Comment (oliverbrooks) [inline README.md:75]: That sounds good. The reader can then see the response and the tests pass :)

**Comment by orangemug** [comment]
@oliverbrooks fixed the issue and updated _readme-tester_ for better error handling.

**Pull request merged**